google/cloud/storage/benchmarks/throughput_experiment.cc

```diff
@@ -165,27 +295,29 @@ std::vector<std::unique_ptr<ThroughputExperiment>> CreateUploadExperiments(
   auto contents = MakeRandomData(generator, options.maximum_write_size);
   gcs::Client rest_client(client_options);
 
-#if GOOGLE_CLOUD_CPP_STORAGE_HAVE_GRPC
-  gcs::Client grpc_client =
-      google::cloud::storage_experimental::DefaultGrpcClient(client_options);
-#endif  // GOOGLE_CLOUD_CPP_STORAGE_HAVE_GRPC
-
   std::vector<std::unique_ptr<ThroughputExperiment>> result;
   for (auto a : options.enabled_apis) {
     switch (a) {
 #if GOOGLE_CLOUD_CPP_STORAGE_HAVE_GRPC
       case ApiName::kApiGrpc:
+      case ApiName::kApiRawGrpc: {
+        gcs::Client grpc_client =
+            google::cloud::storage_experimental::DefaultGrpcClient(
+                client_options);
         result.push_back(
             absl::make_unique<UploadObject>(grpc_client, a, contents, false));
         result.push_back(
             absl::make_unique<UploadObject>(grpc_client, a, contents, true));
-        break;
+      } break;
```

Comment:
Nit: This `break` looks strange sitting on the same line as a closing brace. I'd suggest moving it inside the brace'd block, or at least on its own line following the braced block.

Reply:
meh. Fixed. FWIW, clang-format made me do it :man_shrugging: